config/application.rb

```diff
@@ -15,6 +15,8 @@ class Application < Rails::Application
     # <%= Rails.configuration.councils[ENV['COUNCIL']]['support_email'] %>
     config.councils = config_for(:councils)
 
+    config.autoload_paths += %W(#{Rails.root}/app/models/needs)
```

Comment:
This is just so we can put the need models in their own folder for housekeeping, but is optional.